invenio_app_ils/records/loaders/__init__.py

```diff
@@ -7,7 +7,13 @@
 
 """Invenio App ILS loaders."""
 
+import json
```

Comment:
green light to change `invenio-records-rest` and have `marshmallow_loader` method to accept the error class!